go.mod

```diff
@@ -21,8 +23,11 @@ require (
 	github.com/grpc-ecosystem/go-grpc-prometheus v1.2.0
 	github.com/gtank/merlin v0.1.1
 	github.com/hdevalence/ed25519consensus v0.0.0-20210204194344-59a8610d2b87
+	github.com/lib/pq v1.10.1
 	github.com/libp2p/go-buffer-pool v0.0.2
 	github.com/minio/highwayhash v1.0.2
+	github.com/opencontainers/runc v1.0.0-rc9 // indirect
+	github.com/ory/dockertest v3.3.5+incompatible
```

Comment:
we don't vendor (all of?) our test dependencies, in general, and have kept this file pretty clean. I'm not sure what the actual policy or tactic is here, but I'm doubtful that we really want to vendor `runc` and (most?) of docker? 

Similarly, I'd be curious to know if we're picking up any dependencies that have additional reliance on CGo.

Reply:
dockertest is used to run Docker in-process for testing purposes. We have to either do this or mock it (which is too complex IMHO) or we have to use a real local postgres instance.

Reply:
Sorry I was arguing, weakly, in favor of having a real local postgres instance, and not in favor of mocking.

Reply:
@tessr @cmwaters @marbar3778 thoughts? Should we enforce a local postgres instance to run these tests or use `dockertest` as we are now?

Reply:
Im fine with how it is set up now. It's easier to require docker for tests than installing a db that a dev may not already have, its also can be a pain dealing with local environments. 

Introducing cgo for testing seems fine, the binary won't be built with it from my understanding? Also having cgo in a dep isn't the worst, we already have it in a few places and will most likely introduce it in the future for BLS signatures when they land.

Reply:
I didn't use a real local Postgres instance in the tests because it's too complex for setting up the env. Also considering running repo on a different OS system, might have issues for using local Postgres instance (need more works). 

However, I do find the dependency can be simplified by using `go mod vendor` Will update it in the following commits.
Thanks, @tychoish @alexanderbez @marbar3778

Reply:
I honestly think `dockertest` is fine IMHO.